src/main/scala/rocket/Frontend.scala

```diff
@@ -59,15 +59,15 @@ class FrontendIO(implicit p: Parameters) extends CoreBundle()(p) {
   val perf = new FrontendPerfEvents().asInput
 }
 
-class Frontend(val icacheParams: ICacheParams, hartid: Int)(implicit p: Parameters) extends LazyModule {
+class Frontend(val icacheParams: ICacheParams, staticIdForMetadata: Int)(implicit p: Parameters) extends LazyModule {
```

Comment:
I am missing something fundamental in trying to rename this parameter to better capture its intended use. What is "Metadata" meant to encompass? In a sentence is it meant to say "this is the id for metadata but dont use it to generate hardware"?

Reply:
yeah, once I read more down it made sense. I can resolve this comment.